cachix-api/src/Cachix/Api.hs

```diff
@@ -55,10 +58,10 @@ data BinaryCacheAPI route = BinaryCacheAPI
   , nar :: route :-
       "nar" :>
       Capture "nar" NarC :>
-      StreamResponseGet '[XNixNar, JSON]
+      StreamGet NoFraming OctetStream (ConduitT () ByteString IO ())
   , createNar :: route :-
       "nar" :>
-      StreamBodyMonad '[XNixNar, JSON] AWS :>
+      StreamBody NoFraming OctetStream (ConduitT () ByteString AWS ()) :> -- XNixNar
```

Comment:
Is the cachix-api conduit dependency avoidable?

Reply:
Not that I know of. Same for AWS.